frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -653,6 +620,32 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [NOOBAA_FLAG],
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      id: 'ocsdashboards',
+      section: 'storage',
+      insertBefore: 'persistentvolumes',
+      componentProps: {
+        // t('ceph-storage-plugin~Overview')
+        name: '%ceph-storage-plugin~Overview%',
```

Comment:
I think this should be `OCS Overview` calling it Overview means we are giving an overall storage overview which we are not giving.

@rawagner @bipuladh @yuvalgalanti Opinions?

Reply:
+1

Reply:
got it... I will address it once I get thumbs up from @yuvalgalanti ... have made rest of the changes...

Reply:
We can't change it to `OCS Overview`... so I am leaving it as it is for now...